packages/data-stores/src/index.ts

```diff
@@ -12,6 +12,7 @@ import * as Site from './site';
 import * as User from './user';
 import * as Verticals from './verticals';
 import * as VerticalsTemplates from './verticals-templates';
+import * as AutomatedTransferEligibility from './woocommerce-eligibility';
```

Comment:
The name of the folder should be `atomic-eligibility` or `automated-transfer-eligibility` because that's what it does. There is noting Woo-specific there.

Reply:
I second this, and there are a few occurrences in the code itself as well (e.g. store key for example)

Reply:
That makes sense. I can adjust the naming.